### DISCUSSION TITLE
Export/Send data from Android app to Azure Application Insights

### DISCUSSION THREAD

**Discussion by rzeznickij** [open-telemetry/opentelemetry-java#6936] [Q&A]
With the App Center shutting down next year (https://learn.microsoft.com/en-us/appcenter/retirement), I'm wondering if I'm able to replace the collection and upload/export of custom events, logs and crashes from my android app to Azure Application Insights using this OpenTelemetry SDK or Android version (https://github.com/open-telemetry/opentelemetry-android). Is it possible?

**Comment by breedx-splk** [comment]
Duplicate/related to https://github.com/open-telemetry/opentelemetry-android/issues/716
